openslides/core/static/js/core/site.js

```diff
@@ -764,18 +766,26 @@ angular.module('OpenSlidesApp.core.site', [
             if (!$scope.active_projector) {
                 $scope.changeProjector($scope.projectors[0]);
             }
-
+            $scope.getDefaultProjector();
             // stop ALL interval timer
             cancelIntervalTimers();
 
             rebuildAllElements();
         });
+        $scope.getDefaultProjector = function () {
+            var projectiondefault = ProjectionDefault.filter({name: 'agenda_current_list_of_speakers'})[0];
```

Comment:
renamed to `getDefaultOverlayProjector`. As for the filter, I oriented myself on all the other occurrences of projectionDefault throughout the code (e.g. topics/static/js/topics/site.js:166). I think it would be better to do this in one sepearate issue.